Fixes #28: Remove items from linked lists and reconnect procedure (#1816)
neo4j-contrib/neo4j-apoc-procedures: docs/asciidoc/modules/ROOT/partials/usage/config/apoc.refactor.deleteAndReconnect.adoc

> The procedure support the following config parameters:

Reviewer: ```suggestion
The procedure supports the following config parameters:
```

.Config parameters
[opts=header, cols="1,1,1,5"]
|===
| name | type | default | description

> | relationshipSelectionStrategy | Enum[incoming, outcoming, merge] | true | if `incoming`, the incoming relationship will be attached to the next node, if `outcoming`, the outcoming relationship will be attached, if `merge` will be attacched a merge of incoming and outcoming relatioships. With the same property name, takes precedence the incoming node.

Reviewer: ```suggestion
| relationshipSelectionStrategy | Enum[incoming, outcoming, merge] | true | if `incoming`, the incoming relationship will be attached to the next node. If `outcoming`, the outcoming relationship will be attached. If `merge`, a merge of the incoming and outcoming relationships will be attached. If they share some property names, the incoming relationship properties have higher precedence by default.
```
